### PR TITLE
docs: update contributing instructions

### PR DESCRIPTION
**Pull request by mitchfriedman**
**What this PR does / why we need it**:
Updating the contributing instructions based on [this discussion ](https://kubernetes.slack.com/archives/CK7VC402X/p1583328082001000) on slack.

**Which issue(s) this PR fixes**:
No issue.

@edvald

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,6 +44,13 @@ npm run bootstrap # To bootstrap packages
 
 from the root directory
 
+You will also need to install Node modules in the `garden-service` package.
```
Comment (eysi09) [inline CONTRIBUTING.md]: Shouldn't the `bootstrap` command cover this? If I

```sh
rm -rf garden-service/node_modules && npm run bootstrap
```

...it'll install the `garden-service/node_modules` agin.

Comment (mitchfriedman) [reply CONTRIBUTING.md]: Doesn't that run in the root?

I found that neither `npm install` or `npm run bootstrap` at the root installed the required dependencies in the `garden-service` package that were needed

Comment (eysi09) [reply CONTRIBUTING.md]: That's odd, `npm run bootstrap` (from the root) calls `lerna bootstrap` which in turn runs `npm install` in all packages, that is the `dashboard` and `garden-service`

Comment (eysi09) [reply CONTRIBUTING.md]: Then again: https://github.com/lerna/lerna/issues/1457

Comment (eysi09) [reply CONTRIBUTING.md]: So yeah, I suspect `lerna` is failing silently along the way. I tried some of the suggestions in the issue but that didn't work. 

Could we maybe just rephrase this as:

"You may need to install the Node modules in the `garden-service` package manually due to https://github.com/lerna/lerna/issues/1457."

Comment (mitchfriedman) [reply CONTRIBUTING.md]: Perhaps it was just my machine/environment - do you want to try a clean clone of garden and see if it works without this? If it's not necessary happy to remove this!

Comment (mitchfriedman) [reply CONTRIBUTING.md]: It _could_ have been a versioning thing since I tried to use node 10 from the instructions then realized it was actually using node 12
